src/elements/LicenseIconography/LicenseIconography.vue

```diff
@@ -1,15 +1,22 @@
 <template>
   <span class="vocab license-icons">
-    <FontAwesomeIcon
-      v-for="(icon, index) in processedIconList"
-      :key="index"
-      :icon="['fab', icon]"
-      fixed-width/>
+    <Popup v-bind="$attrs" v-if="showPopup">
+      <FontAwesomeIcon v-for="(icon, index) in processedIconList" :key="index" :icon="['fab', icon]" fixed-width />
+      <template #popup>
+        <Paragraph style="font-size: 1.2599rem">
+          {{ popupContent }}
```

Comment:
Popup content should be passed via a slot which if absent falls back to the default value of the `popupContent` prop.